pkg/asset/internal/templates.go

```diff
@@ -161,7 +169,7 @@ spec:
         - /var/lock/api-server.lock
         - /hyperkube
         - apiserver
-        - --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota
+        - --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds
```

Comment:
quick question: do we have to take over these `admission-control` setting changes also in the Tectonic installer?

Reply:
I cc'd you on an internal issue and will send a follow up email.